Makefile

```diff
@@ -7,6 +7,9 @@ SHELL=/bin/bash
 local: require_virtualenv_in_dev
 	./local_server.sh
 
+start_in_docker:
+	docker run -v `pwd`:/rmc -p 0.0.0.0:5000:5000 -it jgulbronson/uwflow /bin/bash
```

Comment:
@JGulbronson in the `.travis.yml`, you also mount a volume for mongo at `/data/db`. I guess I need that here too, otherwise all of the Mongo data will disappear every time docker shuts down and boots?